services/terrain/src/terrain/services/permanent_id_requests.clj

```diff
@@ -259,16 +259,21 @@
           {:keys [irods-avus metadata]} (metadata-get user folder-id)
           ezid-metadata                 (parse-valid-ezid-metadata irods-avus metadata)
           ezid-response                 (ezid/mint-id shoulder ezid-metadata)
+          identifier                    (get ezid-response (keyword type))
           publish-path                  (publish-data-item user folder (ezid-response->avus ezid-response))]
-      (email/send-permanent-id-request-complete type publish-path))
+      (email/send-permanent-id-request-complete type
+                                                publish-path
+                                                (json/encode ezid-response {:pretty true}))
```

Comment:
Maybe we should return the unparsed EZID API response in the admin email, in case its format changes or our code doesn't parse the new ID correctly for some reason...